app/src/main/java/com/cornellappdev/android/eatery/model/EateryBaseModel.java

```diff
@@ -144,6 +155,32 @@ public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eat
     mArea = CampusArea.fromShortDescription(eatery.campusArea().descriptionShort());
   }
 
+  public void parseCtEatery(Context context, AllCtEateriesQuery.CollegetownEatery ctEatery) {
+    mName = ctEatery.name();
+    mNickName = ctEatery.name();
+    mBuildingLocation = ctEatery.address();
+    mLatitude = ctEatery.coordinates().latitude();
+    mLongitude = ctEatery.coordinates().longitude();
+    mId = ctEatery.id();
+    mPhoneNumber = ctEatery.phone();
+    mEateryType = ctEatery.eateryType();
+
+    List<PaymentMethod> paymentMethods = new ArrayList<>();
+    if (ctEatery.paymentMethods().brbs())
```

Comment:
i don't think you can use brbs for collegetown eateries

Reply:
^^^ same as above, can get rid of but the info is still available